src/logic/basic.lean

```diff
@@ -252,6 +252,7 @@ library_note "decidable namespace"
 protected theorem decidable.not_not [decidable a] : ¬¬a ↔ a :=
 iff.intro decidable.by_contradiction not_not_intro
 
+/-- A version of the law of the excluded middle: the theorem that `¬ ¬ P` is equivalent to `P`. -/
```

Comment:
This should mention the phrase "double negation elimination" somewhere. I'm not sure "a version of" LEM is really accurate. Equivalent to (in the context of...)?

Reply:
Thanks. I have no idea what logicians call these basic concepts. For my entire career up until 2017 they were all vacuous tautologies which didn't need a name.